home/tests.py

```diff
@@ -205,3 +320,33 @@ def tags_response(self, client):
         url = reverse('tags')
         response = client.get(url)
         return response
+
+
+@pytest.mark.django_db
+class TestAddAnswer:
+    def test_add_answer_post(self, client, request, question_test_data, profile):
+        content = {'content': 'test'}
+        request.POST = client.post(f'explore/question_{question_test_data.id}/', data=content)
+        form = CommentForm(request.POST)
```

Comment:
This is a very strange testing technique, its usually better to test views via the client and not try to call then directly as functions. 
Because you're trying to call the view directly you need to create a whole bunch of stuff to pass to if such as that post object that you're creating by crating the form object... this test is actually leaking a whole bunch of internal implementation details....

Wouldn't it be simpler to have the client perform an HTTP post to the view and then inspect the response? See how your team mates did similar thing in their PRs.